packages/typespec-ts/test/integration/generated/azure/core-traits/src/responses.ts

```diff
@@ -33,7 +33,7 @@ export interface SmokeTestDefaultResponse extends HttpResponse {
 
 export interface RepeatableAction200Headers {
   /** Indicates whether the repeatable request was accepted or rejected. */
-  "repeatability-result"?: "accepted" | "rejected";
+  "repeatability-result"?: RepeatabilityResultOutput;
```

Comment:
This is introduced by typespec-azure-core upgrade in which the definition is changed to named union.